src/ProgressiveHedging.jl

```diff
@@ -180,7 +180,7 @@ function solve(tree::ScenarioTree,
                
     timo = TimerOutputs.TimerOutput()
     
-    reset_penalty_parameter(r)
+    r = reset_penalty_parameter(r)
```

Comment:
This makes a copy of the exist data structure.  I'd prefer to reuse the existing one by deleting the needed entries.

Reply:
I changed this because for some reason it wasn't working. I've added a test now to make sure that it does.